PyDSS/value_storage.py

```diff
@@ -601,12 +601,21 @@ def append(self, values):
             list of ValueStorageBase
 
         """
+        #print([v.value for v in values])
+        # try:
         if isinstance(values[0].value, list):
             vals = [x for y in values for x in y.value]
         else:
             vals = [x.value for x in values]
+        # except:
```

Comment:
Can this be removed before merging or do we need to keep it around?

Reply:
removed